### DISCUSSION TITLE
A Gaussian expression for the fine structure constant

### DISCUSSION THREAD

**Discussion by sahil5d** [a-cosmology-group/acg#205] [Astrophysics and Physics]
Here is a candidate expression for the fine structure constant from chip architect Hans de Vries:

http://www.chip-architect.com/news/2004_10_04_The_Electro_Magnetic_coupling_constant.html
archived: https://archive.ph/Q9oB2

http://www.chip-architect.org/physics/fine_structure_constant.pdf
archived: https://archive.ph/QfdoY.

It's a Gaussian value with a convergent, recursive series, and it equals the measured constant to *10 significant digits*.

Here is a simple program to arrive at the value step by step
https://jsfiddle.net/5m1cf7qt.

Knowing that the Hydrogen atom is an eternally reoccurring structure, and that the fine structure constant is essential to the Hydrogen atom, I think it's awesome that the constant is almost certainly of this self-stabilizing form.

This expression was discovered in 2004. That 20 years later this gem still isn't well-known is super strange.

**Comment by RedshiftDrift** [comment]
Interesting, I wasn't aware of this numerical expression for the fine structure constant.  $\alpha$ is essential to the hydrogen atom and also, as explained on the web page, "the number can be seen as the chance that an electron emits or absorbs a photon" so $\alpha$ appears in expressions of the Thomson cross-section, and, more relevant to cosmology, in the expression of the cross section for the stimulated-transfer-redshift mechanism I propose.

What is the physical meaning of this expression?

**Reply by RedshiftDrift** [reply]
Wikipedia doesn't have a link to that article...  Twenty years later, the value still matches measurements!

**Comment by Francois-Zinserling** [comment]
That is an interesting equation and there may be reasoning behind it (I'm not saying I have it!)

If you look at the [equation](https://physics.nist.gov/cuu/Constants/alpha.html) for alpha on CoData

which can also simplifiy to (once again my equation skills are MIA)

α=(q^2)/(2hc*ε0)
(edit, changed charge e to q for clarity and prevent confusion with natural constant e^x)

Point is that this equation is a constant on the left, related to a ratio of constants on the right. Doesn't it mean (at least) one of the constants is redundant?
It could possibly be as a result of our choice of SI units and measures, and thus alpha is really an adjustment of SI? Especially since it has no units.

Or for interpretation, the interpretation from the above links may also have reference to the constants in the right side of the equation, since they essentially make up alpha.

**Reply by sahil5d** [reply]
h might be reunderstood too. I came across this coincidence earlier
`h ≈ proton mass * proton charge radius * c * π/2`.
(https://physics.nist.gov/cgi-bin/cuu/Value?rp)

There's a strong connection between Planck's constant and the proton alone.

Combined with,

- the definition `α = (2pi * kq^2)/(hc)`, and
- the candidate Gaussian expression `α = e^(-pi/^2 / 2) * (1 + α/(2pi)^0 * (1 + α/(2pi)^1 * (1 + α/(2pi)^2 * (1 + ...))))^2`

maybe the SI Units can be rebased on the Hydrogen atom alone, and mathematical constants. The few things pending are connecting the Hubble radius (~ 13.8 Gly), figuring out why it ≈ 2^128 * the electron mass photon wavelength / (2pi), and finding a neat expression for the proton-electron mass ratio.

**Reply by RedshiftDrift** [reply]
@Francois-Zinserling @sahil5d 
> It could possibly be as a result of our choice of SI units and measures, and thus alpha is really an adjustment of SI?

This cannot be the case since readjusting the definitions of the SI units will affect at least two of the physical constants in the equation for alpha.  alpha remains unchanged since the units cancel in the combination of physical constants.  A different choice of SI units would change the numerical value of h, but in the end any ratio that does not have physical units will remain the same. (That all comes out of dimensional analysis, as we discussed earlier with @HanDeBruijn .)

The reasoning behind those 'pure' constants (unitless) and how to get their values ($\pi$, $2^{128}$ or $\alpha$...) is not known in most cases.

**Reply by sahil5d** [reply]
What do you mean the reasoning behind π isn't known?
@RedshiftDrift

**Reply by RedshiftDrift** [reply]
> What do you mean the reasoning behind π isn't known?

Ratio of the mass of the proton to the mass of the electron is $6\pi^5$, why the pure constant $\pi$ appears there is not known.

**Reply by HanDeBruijn** [reply]
>Ratio of the mass of the proton to the mass of the electron is

$$
m_p/m_e = 1836.15267343 \ne 1836.118110 = 6\pi^5
$$

**Reply by sahil5d** [reply]
Oh ok. That proton-electron mass ratio candidate expression isn't near likely true in my opinion. Neither precise, nor uniquely constrained, nor with any critical features...

**Reply by RedshiftDrift** [reply]
... but maybe with corrections of the same type as those used on $\alpha$ you could get the numbers to agree...

**Reply by sahil5d** [reply]
I think the proton-electron mass ratio is more likely to involve α itself.

Also, α relates a lot of radiuses, and the mass ratio is about mass, which is more like volume, so `m_p / m_e` might involve α cubed, or some variation of that.

log(1836.15) / log(137.036) ≈ 1.5
https://www.wolframalpha.com/input?i=log%281836.15%29+%2F+log%28137.036%29

Also, recall the Big Coincidence https://medium.com/@sahil50/a-large-numbers-coincidence-299cb199e03d. That it approximately equals 2^128 is interesting, but there is something else. There is another relation to be pondered. Why is the bottomleft/topright what it is? 
`m_p / 4 / m_e`. 
Is `m_p / 4 / m_e` ≈ 459.038 something important in itself?

**Comment by HanDeBruijn** [comment]
Okay. I've made some exercises with [MAPLE 8](https://en.wikipedia.org/wiki/Maple_(software)) and my outcomes are **not** in complete agreement with the claims as made in this thread. I've started with a $\Gamma$ series of order 7 but it turns out that the end result for $\alpha$ is not essentially different if this is reduced to a $\Gamma$ series of order 2. Then we get the following with our favorite Computer Algebra System.
```
reeks := 1+alpha*(1+alpha/(2*Pi)*(1+alpha/(2*Pi)^2));
fsolve(reeks^2*exp(-Pi^2/2)-alpha=0,alpha);

                     0.007297352558, 11.56783268
```
Yes of course: in general there is more than one solution. But only the first one ($= 0.007297352558$) is remarkably insensitive to the order of the $\Gamma$ series. However, as far as I can see, it does **not** exactly converge to the _current estimate of the fine-structure constant_ (on the right): 

$$
0.007297352558 \ne 0.0072973525693
$$

**Reply by sahil5d** [reply]
It might be an issue with Maple's fsolve or float precision https://www.maplesoft.com/support/help/maple/view.aspx?path=fsolve.

I tried the Maple fsolve value, the actual fine structure constant, and the De Vries iterated value in your fsolve argequation, and you can see here which is the closest to zero, https://jsfiddle.net/b5c1dofz.

**Reply by HanDeBruijn** [reply]
@sahil5d . You are quite right. The calculation can easily be improved by asking for more digits. As follows.
```
Digits := 11;
reeks := 1+alpha*(1+alpha/(2*Pi)*(1+alpha/(2*Pi)^2));
fsolve(reeks^2*exp(-Pi^2/2)-alpha=0,alpha);

                     0.0072973525686, 11.567832669
```
The difference between 0.0072973525686 and 0.0072973525693 has now become 0.0000000000007.
We can even go for the 21 significant digits at the bottom of the referenced [paper](http://www.chip-architect.org/physics/fine_structure_constant.pdf): &alpha; = 0.00729735256865385342269 .
```
Digits := 21;
reeks := 1+alpha*(1+alpha/(2*Pi)*(1+alpha/(2*Pi)^2*(1+alpha/(2*Pi)^3*(1+alpha/(2*Pi)^4))));
fsolve(reeks^2*exp(-Pi^2/2)-alpha=0,alpha);

                     0.00729735256865385342275, 11.5047358071896377852
```
Question remains: has the outcome of the mathematics a **physical** meaning?!

More  about Hans de Vries [here](https://nl.linkedin.com/in/hans-de-vries-0156206) . Two websites: [Chip-Architect](http://www.chip-architect.com/) , [Physics Quest](http://www.physics-quest.org/) .

**Reply by sahil5d** [reply]
> has the outcome of the mathematics a physical meaning?!

I don't know the perfect physical meaning, but it has to be related to the shape of the electron-mass gamma ray photon, the shape of the electron, and the shape of the 1s electron orbital.

The e^(-pi^2/2) part of the fine structure constant expression is similar to the radial probability distribution function of the 1s orbital.

<img width="387" alt="Screenshot 2024-03-16 at 12 12 03 PM" src="https://github.com/a-cosmology-group/acg/assets/5514802/cd7de083-d8d2-4219-8dbd-88eef65fe91c">

(https://youtu.be/Nr40fnfHccQ?si=d6AjbomxO8yO2RZp&t=41)

**Comment by genotics** [comment]
I am pleased to see that the alpha enigma is again under scrutiny. You may recall that I have referred to the work of the late Malcolm Mac Gregor in past posts. Malcolm spent the last 30 or so years of his life developing a model of an extended electron structure and refining and expanding the range of application of the alpha constant, all of which is summarized in 5 books published over that period, the last of which, the Alpha Sequence, appeared posthumously, a project to which he devoted the last year of his life. 
In 2016-17 we engaged in an unfinished research project to link the extended electron to the alpha constant based on a hypothesis put forward by Raji Heyrovska whereby the fine structure constant is  obtained from the Golden Angle in the hydrogen atom. I append below Malcolm's assessment of Heyrovska's thesis and his assessment  of further research prospects.
My own appreciation of these prospects rests on the notion that Malcolm's relativistically rotating sphere model of the electron, essentially a geometrical construct, must be replaced in the process of extending the range of the fine structure constant, now perceived as an angle of rotation, by a vortex model, which would also entail casting the vacuum as a fluid medium. This in turn might open new vistas in the understanding of elementary particles and interactions, perhaps also including the redshift phenomenon.
Roy
Here is Malcolm's final assessment:

The papers of Browne, Broberg and Martin describe particle models that attempt to extend the information about the α quantized data base of elementary particle lifetimes and mass/energies so as to include the surrounding vacuum state and the mechanisms that give rise to the creation and properties of these particles. The paper of Heyrovska and Narayan and other papers by her group start with the elementary particles themselves and demonstrate how they combine in atomic, molecular and solid state configurations that can be understood and predicted with the aid of Golden Mean representations.
The ground-state orbit in the Bohr hydrogen atom is used to illustrate the systematics of the Golden mean phenomenology, and the application of the Golden ratio ϕ power series to other systems. The circular Bohr radius rB is a factor of 137 larger that the Compton radius, and the electron velocity in the Bohr radius is a factor of 137 slower than the velocity of light c. Thus these properties of the Bohr model are dictated by the values of the constants e, h, and c that make up the fine structure constant  Heyrovska demonstrates that the Bohr radius, which is the separation distance between the proton and electron in the Bohr hydrogen atom, can be divided into two Golden mean segments, and she describes the graphical representation to carry out this division. The Golden segments can be evaluated as terms in the power series expansion of the Golden Ratio ϕ= 1.618, whose mathematical properties have fascinated scientists for centuries. The power series in ϕ can in turn be related to the Fibonacci Series, which has broad applications in many areas. (See, for example, the paper Fibonacci Numbers and the Golden Ratio in Biology, Physics, Astrophysics, Chemistry and Technology: A Non-Exhaustive Review, by Vladimir Pletser, Chinese Academy of Sciences, Beijing, China.)

Heyrovska has expanded the Golden Ratio analysis of the hydrogen atom to include the entire hydrogen spectral line data base for experimental determinations of the fine structure constant and the Rydberg constant. She has also applied the Golden Radius analysis to a determination of ionization bond lengths, as specified in the title of a Vixra report:  Bohr Radius as the Sum of Golden Sections Pertaining to the Electron and Proton, Covalent Bond Lengths Between Same Two Atoms as Exact Sums of Their Cationic and Anionic Radii and Additivity of Atomic and or Ionic Radii in Bond Lengths.

The interesting aspect of the Heyrovska Golden Ratio formalisms for our purposes is to see if they can be applied to the production and properties of the elementary particles themselves, which is an area that she has not addressed.

Next area for The 137 Project to investigate: The Thomson radius
The Thomson radius or Thomson scattering length, which is identical to the classical electron radius, is experimentally determined by the low-energy scattering of electromagnetic photons off electrons, where the photon wavelength is much longer than the size of the electron, and where the photon energy is too weak to dislodge the electron from its orbit in an atom. In this low-energy limit, the Thomson scattering cross section is independent of the photon energy, and thus represents the basic coupling between an electromagnetic beam and the charge on a stationary electron. The Thomson radius is a factor of 137 smaller than the electron Compton radius, which in turn is a factor of 137 smaller than the Bohr radius. 
If we picture the first step in particle production as being the creation of an electron-positron pair out of a pair of Thomson-sized Coulomb energy packets, then our task is to see if the Heyrovska Golden Ratio formalism can be usefully applied to these energy packets. If particle production is actually expressible in terms of Golden Ratio concepts, it might be revealed most clearly at the low-energy threshold for producing particles out of electromagnetic energy.

**Comment by genotics** [comment]
Here is further detail from the abstract of  Malcolm's last published paper.

Abstract: Precision Tevatron and Linear Hadron Collider measurements at Fermilab and CERN have revealed the
numerically accurate mass equality W ? Z = t. This equality between two gauge bosons (gb) and the top quark t is only
valid if reinterpreted as an energy equality, where E = mc2, since energy is a shared property of particles and quarks. The
experimental data indicate that the LHC particle excitation energy is quantized in the form of gauge boson energy packets
Egb, which are created by factor-of-137 proton-quark energy increases denoted as a-boosts, where a * 1/137 is the fine
structure constant. These a-boosts occur during the rare head-on quark–quark collisions in the proton beams. The a-boost
energy quantization mechanism also occurs in low-energy electron–positron boson and fermion particle production
channels, where it generates Eb and Ef energy packets. These a-boost energy channels link together coherently, as
demonstrated by the accurate top quark energy equation Etop = (18/a2) Eelectron. Particle production energy equations are
derived which combine to create an overall energy pattern that accurately reproduces the energies of the (u,d), s, c, b,
t fermion constituent quarks, the l and s leptons, and the proton.
